tests/coreos-install/positive/general.go

```diff
@@ -97,6 +96,29 @@ func init() {
 	})
 }
 
+// used by tests which want to test versions without pinning
+// a specific channel as on Container Linux machines the
+// channel will default to the host machines channel.
+//
+// Will update the test.Version flag with the selected Version
+func pickVersion(t *testing.T, test register.Test) {
+	pinnedVersions := map[string]string{
+		"alpha":  "1535.0.0",
+		"beta":   "1520.3.0",
+		"stable": "1465.7.0",
+	}
+
+	channel, _, _ := util.GetDefaultChannelBoardVersion(t)
```

Comment:
This commit no longer builds, since the arg/return fixup was added to the next commit instead.

Reply:
Whoops, backporting the fix into the proper commit

Reply:
Backported into the `tests/coreos-install: update version only test on cl machines` commit and pushed, good to merge on my end.